docs/source/dev/analytics/jupyter.rst

```diff
@@ -3,83 +3,35 @@
 Jupyter Notebook
 ================
 
-Jupyter is a development tool that allows users to create notebooks containing comments and code, like iPython Notebook. Jupyter supports other languages via the use of 'kernels'.
+Jupyter is a web application that allows users to create and share documents containing live code,
+equations, visualizations and narrative text.  This section describes how to configure a Jupyter Python 3 notebook
+to allow access to Koverse data sets.
 
-To use Jupyter with Koverse and PySpark, first create a kernel.json file in a folder called 'koverse'
+The prerequisites for accessing Koverse data sets in a Jupyter Python 3 notebook are::
 
-Configure the kernel.json file as follows by setting the right value for SPARK_HOME::
+  Spark 1.6 installed as /opt/spark
```

Comment:
I think we could leave out the requirement of installing it in /opt/spark

Reply:
Oh I see the env variable reference for SPARK_HOME below. I don't want the user to think it has to go in /opt/spark but I do see the value in providing an example here like you have.